openbas-front/src/utils/Localization.js

```diff
@@ -846,7 +846,7 @@ const i18n = {
       Friday: 'Vendredi',
       Saturday: 'Samedi',
       Sunday: 'Dimanche',
-      Hour: 'Heure',
+      Scheduling_time: 'Heure',
```

Comment:
Why not put it directly as a **time** label in the code? to avoid additional translation at the trad file level

Reply:
The label 'time' is already used in the code with another french translation.